### PR TITLE
Added CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Added a contributor's guide for this repository.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

---
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-ios/99)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Contributing to RudderStack
+
+Thanks for taking the time and for your help in improving this project!
+
+## Table of contents
+
+- [**RudderStack Contributor Agreement**](#rudderstack-contributor-agreement)
+- [**Contribute to this project**](#contribute-to-this-project)
+- [**Submitting a pull request**](#submitting-a-pull-request)
```
Comment (pallabmaiti) [inline CONTRIBUTING.md:9]: @ameypv-rudder we don't have description regarding this section below. Can you add some description below?

Comment (ameypv-rudder) [reply CONTRIBUTING.md]: Sure - fixed this.

**Review by pallabmaiti — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,14 +1,38 @@
...
+    <a href="https://rudderstack.com/join-rudderstack-slack-community">Slack</a>
+  </b>
+</p>
+
+---
 
 # RudderStack iOS SDK
 
-RudderStack's iOS SDK lets you track event data from your iOS applications. After integrating the SDK, you will be able to send the event data to your preferred destination/s such as Google Analytics, Amplitude, and more.
+RudderStack's iOS SDK lets you track event data from your iOS applications. After integrating the SDK, you will be able to send the event data to your preferred destinations such as Google Analytics, Amplitude, and more.
```
Comment (arnab-p) [inline README.md:29]: Let's mention tvOS as well here

File: README.md [hunk 3]
```diff
@@ -32,6 +56,46 @@ github "rudderlabs/rudder-sdk-ios" "v1.1.4"
...
+```swift
+import PackageDescription
+
+let package = Package(
+    name: "RudderApp",
+    dependencies: [
+        // Add a package containing Analytics as the name along with the git url
+        .package(
+            name: " ",
+            url: ""
```
Comment (arnab-p) [inline README.md:87]: @pallabmaiti Should we prefill the URL here?

**Review by arnab-p — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (pallabmaiti) [reply README.md:87]: Yes. I think they are still woking on it.

File: README.md (hunk 2 above)
Comment (ameypv-rudder) [reply README.md:29]: sure

File: README.md (hunk 3 above)
Comment (ameypv-rudder) [reply README.md:87]: @arnab-p this is incomplete. I'm waiting for some inputs from Abhishek before we proceed. Need to add screenshots as well.

Comment (ameypv-rudder) [reply README.md:87]: I'll ping both of you once the PR is ready to review. Changed the status to WIP for now.

**Review by 1abhishekpandey — APPROVED** [review]

**Review by pallabmaiti — APPROVED** [review]

**Pull request merged**